src/template_vol_connector.c

```diff
@@ -24,7 +24,7 @@
 
 /* The VOL class struct */
 static const H5VL_class_t template_class_g = {
-    2,                                              /* VOL class struct version */
```

Comment:
I believe it may be more correct to update this to "3" rather than using the H5VL_VERSION macro.